.travis.yml

```diff
@@ -15,9 +15,9 @@ notifications:
 
 env:
   matrix:
-    - ROS_DISTRO=melodic  ROS_REPO=ros
-    - ROS_DISTRO=melodic  ROS_REPO=ros-testing
-    - ROS_DISTRO=melodic  UPSTREAM_WORKSPACE='https://raw.githubusercontent.com/ros-controls/ros_control/$ROS_DISTRO-devel/ros_control.rosinstall -ros_controllers'
+    - ROS_DISTRO=noetic  ROS_REPO=ros
+    - ROS_DISTRO=noetic  ROS_REPO=ros-testing
+    - ROS_DISTRO=noetic  UPSTREAM_WORKSPACE='https://raw.githubusercontent.com/ros-controls/ros_control/$ROS_DISTRO-devel/ros_control.rosinstall -ros_controllers'
```

Comment:
Shouldn't this [line](https://github.com/ros-controls/ros_control/blob/fc5a5d7f4df57de201ae5d97df42f2d754faf1a8/ros_control.rosinstall#L2) be uncommented?

Reply:
I'm pretty sure you are right, nice catch!

https://github.com/ros-controls/ros_control/pull/475

Reply:
Ah... the PR I tried now tries to build controllers prior to building ros_control :/